---
title: "geneSummaryVign"
author: "Nicholas Hutson"
date: "`r Sys.Date()`"
output: rmarkdown::html_vignette
vignette: >
  %\VignetteIndexEntry{geneSummaryVign}
  %\VignetteEngine{knitr::rmarkdown}
  %\VignetteEncoding{UTF-8}
---

```{r, include = FALSE}
knitr::opts_chunk$set(
  collapse = TRUE,
  comment = "#>"
)
```

```{r setup}
library(geneSummary)
```

## Shiny
A shiny app that allows for data to be imported from the NCBI GEO database. Different data tables can be generated and saved.

## Extract expression matrix (extExp)
This function extracts the numerical gene expression data. It is in BioConductor format with genes as rows and samples as columns.

## Extract sample matrix (extSample)
This function extracts the annotations for each sample. This is information about each sample such as sample characteristics and cell type.

## Extract gene annotation (extGene)
This function extracts the annotations for each gene. This is information about the measurement of each gene such as sequencing of the gene and probe ID. 

## Visualization (box and bar)
These functions visualize the concentration of the gene TP53. Bar creates a histogram to compare the concentration between samples and box creates a box plot to compare the concentration between the two groups of samples. 

## Comparison and histogram
These functions creates a table and histogram from the gene concentration for the ten genes with the greatest variances between samples and TP53. The table compares the mean, standard deviation, and median between the two groups of samples and the total gene expression data for each gene. The histogram compares the average gene expression for each of the genes and includes error bars.  
