---
title: "geneSummaryVign"
author: "Nicholas Hutson"
date: "6/3/2019"
output: rmarkdown::html_vignette
vignette: >
  %\VignetteIndexEntry{geneSummaryVign}
  %\VignetteEngine{knitr::rmarkdown}
  %\VignetteEncoding{UTF-8}
---

```{r, include = FALSE}
knitr::opts_chunk$set(
  collapse = TRUE,
  comment = "#>"
)
```

```{r setup}
library(geneSummary)
```

## Shiny
A shiny app that allows for data to be imported from the NCBI GEO database. Different data tables can be generated and saved.

## Extract expression matrix (extExp)
This function extracts the numerical gene expression data. It is in BioConductor format with genes as rows and samples as columns.

## Extract sample matrix (extSample)
This function extracts the annotations for each sample. This is information about each sample such as sample characteristics and cell type.

## Extract gene annotation (extGene)
This function extracts the annotations for each gene. This is information about the measurement of each gene such as sequencing of the gene and probe ID. 

## Filter data tables (filterTbl)
This function filters data tables by values in each of their columns.
